### PR TITLE
DOCSP-25800 command line connection support

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING - cmd line connect](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-25800-commandline-connection-support/connect/connect-from-the-command-line/)
[STAGING - connect](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-25800-commandline-connection-support/connect/#connecting-with-compass-from-the-command-line)

[JIRA](https://jira.mongodb.org/browse/DOCSP-25800)

### CODE REVIEW

File: source/connect/connect-from-the-command-line.txt [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+If your connection string contains sensitive information, consider
+using a configuration file to avoid exposing that information on
+the command line.
+
+.. _compass-executable-location:
+
+Compass Executable Location
+---------------------------
+
```
Comment (jmd-mongo) [inline source/connect/connect-from-the-command-line.txt:32]: [praise]

Thanks for pointing out this info in a clear and accessible way!

File: source/connect/connect-from-the-command-line.txt [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+ 
+   mongodb-compass mongodb+srv://cluster0.cpxklfq.mongodb.net/library
+ 
+
+.. _compass-connect-file-based:
+
+Configuration File Connection Specification
+-------------------------------------------
+
+The command line invocation for |compass-short| can include a reference
```
Comment (jmd-mongo) [inline source/connect/connect-from-the-command-line.txt:85]: [optional suggestion]

Consider "...can specify a configuration file". I think this is slightly more specific and is consistent with the header.

File: source/connect/connect-from-the-command-line.txt [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+to a configuration file. 
+
+The format is:
+
+.. code-block:: shell
+ 
+   <path/to/compass/executable> \
+      --file=<filename> \
+      [--passphrase=<passphrase>] \
+      [<connection id>]
```
Comment (jmd-mongo) [inline source/connect/connect-from-the-command-line.txt:95]: [question] 

Should "id" be in caps?

File: source/connect/connect-from-the-command-line.txt [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+The components of the command invocation are:
+
+- The path to the |compass-short| executable
+- A connection configuration file
+- An optional passphrase for the connection configuration file
+- An optional connection id
```
Comment (jmd-mongo) [inline source/connect/connect-from-the-command-line.txt:102]: Same q as on line 95.

File: source/connect/connect-from-the-command-line.txt [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+To open |compass-short| and connect to your MongoDB instance, use a
+command line like: 
+
+.. code-block::
+   :copyable: false
+
+   mongodb-compass --file=learningConnectionFile \
+                   --passphrase=superSecret
+
+If you have multiple favorites, include the connection id from the
```
Comment (jmd-mongo) [inline source/connect/connect-from-the-command-line.txt:124]: Same q as on line 95.

**Review by jmd-mongo — COMMENTED** [review]
Thanks for the updates, @davemungo! I've left a suggestion and a few qs. I think that [this concurrent issue](https://jira.mongodb.org/browse/DOCSP-25363) may cover some of the same ground, just FYI. 

Thanks!
Joe

**Review by davemungo — COMMENTED** [review]
@jmd-mongo  thanks for the feedback, this is ready for another pass. 

Yes they are related tickets. There are one or two other as well. I think the other one is more about the config file itself. If it goes in first We can link to it from here.

**Review by jmd-mongo — APPROVED** [review]
Thanks for the changes here, @davemungo! This one LGTM at this point.

Thanks!
Joe

**Review by addaleax — APPROVED** [review]

**Pull request merged**